db/institutions.yml

```diff
@@ -285,3 +285,12 @@ utoronto:
   state: Ontario
   street: 27 King's College Cir
   zip: M5S 3H7
+umontreal:
+  aka: [UdeM, udem]
```

Comment:
could probably have some anglicisations.  If someone called it Montreal or University of Montreal or University Montreal it wouldn't be found.